doc/classes/Area2D.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		[Area2D] is a region of 2D space defined by one or multiple [CollisionShape2D] or [CollisionPolygon2D] child nodes. It detects when other [CollisionObject2D]s enter or exit it, and it also keeps track of which collision objects haven't exited it yet (i.e. which one are overlapping it).
 		This node can also locally alter or override physics parameters (gravity, damping) and route audio to custom audio buses.
+		[b]Note:[/b] Areas and bodies created with [PhysicsServer2D] might not interact as expected with [Area2D]s, and might not emit signals or track objects correctly.
```

Comment:
[b]Note:[/b] [Area2D]s and [PhysicalBody2D]s created..

Also, is this correct? This makes it sound like creating them from PhysicsServer would result in instability. Is this only at the very beginning of simulation or?

Reply:
This is correct, they are not associated with a node so they work differently,